.github/workflows/update-submodules.yml

```diff
@@ -49,8 +49,13 @@ jobs:
           SUBMODULE: ${{ github.event.inputs.repo }}
           BRANCH: ${{ steps.target_branches.outputs.submodule_branch }}
         run: |
+          if [[ "$SUBMODULE" != "temporal" && "$SUBMODULE" != "tctl" ]]
```

Comment:
Maybe do:
```
if ! git config --file=.gitmodules --get submodule."$SUBMODULE".url >/dev/null; then
  echo "unknown submodule $SUBMODULE"
  exit 1
fi
```
so it's using the same source of truth about which submodules exist?